i18n/fr/docusaurus-plugin-content-docs/version-22.04/releases/centreon-core.md

```diff
@@ -456,6 +456,34 @@ By:
 
 ## Centreon Collect
 
+### 22.04.4
+
+Release date: `soon`
+
+#### Centreon Broker
+
+##### Enhancements
+
+- Added a new gRPC API endpoint to modify the flush period of Engine's and Broker's loggers without reloading or restarting the services.
```

Comment:
This enhancement affects both engine and broker (it should be duplicated to the engine section imo)